README.md

```diff
@@ -370,6 +371,22 @@ The following example connects to multiple exchanges:
 
 * `$ rabtap tap my-fanout-exchange:,my-topic-exchange:#,my-other-exchange:binding-key`
 
+##### Tap all messages published or delivered 
+
+The [RabbitMQ Firehose Tracer](https://www.rabbitmq.com/firehose.html) allows to "see" every message that
+is published or delivered. TO use it, the firehose tracer has to be enabled first:
+
+```shell
+$ rabbitmqctl rabbitmqctl trace_on 
+```
+
+Then every message published or delivered will be CC'd to the topic exhange `amq.rabbitmq.trace`. 
+At this exchange, the messages can now be tapped with rabtap:
+
+```shell
+$ rabtap tap amq.rabbitmq.trace:#`
```

Comment:
2 things:
1. the last tick (`) should be removed?

2. Why not using the default user name & password so it comes 🔋 included for noobs (like me!) ?
```
rabtap tap --uri amqp://guest:guest@localhost:5672/ amq.rabbitmq.trace:# 
```

Reply:
Ok, will add the --uri option, however I recommend using the RABTAP_AMQPURI environment variable, making things easier